src/label-for-moment/label-for-moment.js

```diff
@@ -0,0 +1,26 @@
+import _ from 'lodash';
+
+export default function labelForMoment(date) {
+  var date = new Date(date);
```

Comment:
it would have been much better to actually use the moment object which has some tools to do these things, rather than converting it to a date (or to at least have asked why we were using moment and not date if you wanted to use date), but in the interest of time, we can leave as you have it